### ISSUE TITLE
Debugger crashes ungracefully if one tries to debug invalid syntax

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/julia-vscode#1399]

**Comment by pfitzseb** [comment]
Could potentially try to parse the whole file and show an error if that fails.

**Comment by davidanthoff** [comment]
Yes, I think that would be the right strategy. We might just need an exception handler at https://github.com/julia-vscode/DebugAdapter.jl/blob/4c1432beee49d97f9281926acf4b27fbbd50dd16/src/debugger_requests.jl#L22 that then returns a JSONRPC error that we handle in the client.
